packages/permissions/src/components/authorized/authorized.tsx

```diff
@@ -76,64 +75,47 @@ const defaultProps: Pick<Props, 'shouldMatchSomePermissions'> = {
 };
 
 const Authorized = (props: Props) => {
-  // Check first for data fences
-  if (
-    props.actualDataFences &&
-    props.demandedDataFences &&
-    props.demandedDataFences.length > 0
-  ) {
+  // if the user has no permissions and no dataFences assigned to them, they are not authorized
+  if (!props.actualDataFences && !props.actualPermissions) {
```

Comment:
this case won't happen since `actualDataFences` and `actualPermissions` won't be `undefined`, right?

Reply:
actualDataFences can be, and if we look at the old code we where doing this: 
```
if (!props.actualPermissions)	
    return <React.Fragment>{props.render(false)}</React.Fragment>;
```

so also actual permissions can be `null`

Reply:
👍 thanks